### PR TITLE
fix: quantity input of additional products have an issue after changing quantity of a product in the cart

### LINKED ISSUE #540: [Shopping Cart] quantity input of additional products have an issue after changing quantity of a product in the cart
<h3>Describe the bug and add attachments</h3>
<p>In the shopping cart when you change the quantity of a product, the products displayed below the cart have problem when you change their quantity.</p>

https://github.com/PrestaShop/hummingbird/assets/110676325/708cafdc-c3c1-4d92-b54d-ba63c12395c5

<h3>Expected behavior</h3>
<p>The quantity input of additional products should increment 1 by 1</p>

<h3>Steps to reproduce</h3>
<ol>
<li>FO -> Add product to cart</li>
<li>Change the quantity of the product in your cart</li>
<li>Change the quantity of an additional product which appear below your cart</li>
</ol>

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Correct the issue 540
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/hummingbird/issues/540
| Sponsor company   | @PrestaShopCorp
| How to test?      | You can refer to the issue

### CODE REVIEW

**Comment by Hlavtox** [comment]
Seems that built assets got into the PR

**Review by kpodemski — DISMISSED** [review]
Compiled assets shouldn't be versioned.

**Comment by tblivet** [comment]
The compiled files are removed !

**Review by ancardinale — APPROVED** [review]

**Comment by ga-devfront** [comment]
No response from the people who blocked the PR and the changes are be done.
I pass the PR to Waiting for QA

**Review by ga-devfront — DISMISSED** [review]

**Review by ancardinale — APPROVED** [review]

File: src/js/constants/selectors-map.ts [hunk 1]
```diff
@@ -105,6 +105,7 @@ export const desktopMenu = {
 
 export const qtyInput = {
   default: '.js-quantity-button',
+  productCartList: '.cart__items .js-quantity-button',
```
Comment (davidglezz) [inline src/js/constants/selectors-map.ts:108]: Looks like this is not a good place

It contains selectors of `qtyInput` parts, not where `qtyInput` is placed

maybe inside `cart` const? `cart.productQuantity`

**Review by djoelleuch — DISMISSED** [review]
Hello @tblivet  
LGTM ,

https://github.com/PrestaShop/hummingbird/assets/115152460/2910e1fa-ab18-4ec0-b52d-693cc618f6d0




It's QA approved ✔️
thank you !

**Review by ga-devfront — APPROVED** [review]

**Pull request merged**